ui/admin/tests/acceptance/credential-store/read-test.js

```diff
@@ -81,4 +81,14 @@ module('Acceptance | credential-stores | read', function (hooks) {
       'Error 404'
     );
   });
+
+  test('Users can link to docs page for credential store', async function (assert) {
+    assert.expect(1);
+    await visit(urls.credentialStore);
+    assert.ok(
+      find(
+        `[href="https://boundaryproject.io/help/admin-ui/credential-stores"]`
```

Comment:
A bit late to the party (I saw already merged).

I am fine with this test. If we want to test the user reach certain URL there is no other way but comparing that.

That said, since we already have a pattern to define urls (line 20 `const urls`) I will suggest to add this URL there, something like:
```javascript
const urls = {
  ...
  docsCredentialStore: 'https://boundaryproject.io/help/admin-ui/credential-stores'
}
```

Then on the test you just reference the url within the assertion.

I think that small change can facilitate mantainability of the code and facilitat (if need) re-using the code.